server/repository/src/db_diesel/changelog/test.rs

```diff
@@ -154,7 +181,7 @@ async fn test_changelog_iteration() {
         .unwrap();
 
     // test iterating through the change log
-    let changelogs = repo.changelogs(0, 3, None).unwrap();
+    let changelogs = repo.changelogs(3, 3, None).unwrap();
```

Comment:
why not start at 0? or was there any reason to change it?

Reply:
Assert was checking location id... I can start at 0 and check name id instead?

Reply:
was there a problem with `(0, 3, None)` shouldn't both work?

Reply:
If you start from 0, it will look up the name changelogs, starting from 3 is the location changelogs... do you want me to start from 0?

Reply:
No sorry my bad, misunderstood it